apps/examples/nettest/nettest.c

```diff
@@ -261,6 +261,71 @@ void broadcast_send(uint32_t num_packets, uint32_t sleep_time)
 	close(sock);
 }
 
+/* ------------------------------------------------------------ */
+/*                                                              */
+/* Receive Broadcast example.                                   */
+/*                                                              */
+/* ------------------------------------------------------------ */
+void broadcast_receive(uint32_t num_packets)
+{
+	int sockfd;
+	int is_broadcast = 1;
+	int count;
+	int nbytes;
+	char databuf[256];
+	struct sockaddr_in recvaddr;
+	struct sockaddr_in sendaddr;
+	socklen_t addrlen = sizeof(struct sockaddr_in);
+
+	printf("[BRSERVER] broadcast receive start!\n");
+
+	sockfd = socket(AF_INET, SOCK_DGRAM, 0);
+	if (sockfd < 0) {
+		printf("[BRSERVER] socket create fail err(%d)\n", errno);
+		return;
+	}
+
+	if (setsockopt(sockfd, SOL_SOCKET, SO_BROADCAST, &is_broadcast, sizeof(is_broadcast)) < 0) {
+		printf("[BRSERVER] set socket option fail err(%d)\n", errno);
+		goto return_with_close;
+	}
+
+	memset(&recvaddr, 0, sizeof(recvaddr));
+	recvaddr.sin_family = AF_INET;
+	recvaddr.sin_addr.s_addr = htonl(INADDR_ANY);
+	recvaddr.sin_port = htons(g_app_target_port);
+
+	if (bind(sockfd, (struct sockaddr *)&recvaddr, sizeof(recvaddr)) < 0) {
+		printf("[BRSERVER] bind fail err(%d)\n", errno);
+		goto return_with_close;
+	}
+
+	count = 0;
+
+	while (1) {
+		memset(databuf, 0, sizeof(databuf));
+
+		nbytes = recvfrom(sockfd, databuf, sizeof(databuf), 0, (struct sockaddr *)&sendaddr, (socklen_t *)&addrlen);
+		if (nbytes <= 0) {
+			printf("[BRSERVER] receive data fail err(%d)\n", errno);
+			goto return_with_close;
+		}
+		count++;
+		printf("[BRSERVER] received data from %s #%d : %s\n", inet_ntoa(sendaddr.sin_addr), count, databuf);
+
+		if (num_packets == 0) {
+			continue;
+		}
+		if (count >= num_packets) {
```

Comment:
How about below instead of 2 if states? count is always greater than 1
if (num_packets != 0 && num_packets <= count) {
    break;
}

Reply:
I will change all same things in this file via another commit